thesis-api/main/services/category_service.py

```diff
@@ -14,5 +15,14 @@ def __init__(self, dao: CategoryDAO):
     def build(db: Session):
         return CategoryService(CategoryDAO(db))
 
+    def create_all(self, fn: str):
+        with open(fn, mode='r') as csv_file:
+            csv_reader = csv.DictReader(csv_file, delimiter=';')
+            for row in csv_reader:
```

Comment:
Konrad stosował konwencję add_all i nie wiem czy się jej trzymamy czy nie, ale to lepiej niech się Konrad wypowie, tak poza tym to jest ok

Reply:
Ok, o mogę zmienić, żeby było jednolicie